.github/workflows/cloudflare_pages.yaml

```diff
@@ -0,0 +1,29 @@
+# This creates PR previews like https://<branch-name>.obsidianirc.pages.dev
+# From main it should deploy to https://obsidianirc.pages.dev
+
+name: Deploy to Cloudflare Pages
```

Comment:
This will only deploy from main to https://obsidianirc.github.io/

We could use that page for something but for now I guess it is ok. This workflow is pushing to the repo https://github.com/ObsidianIRC/obsidianirc.github.io